### PR TITLE
[ClassContent] updated and fixed revert and commit of contents drafts

### PR DESCRIPTION
**Pull request by eric-chau**
Commit and revert of content's draft does not work fine sometimes. This PR aims to fix it.

ping @crouillon

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
this need a rebase, sorry @eric-chau

File: ClassContent/AbstractContent.php [hunk 1]
```diff
@@ -503,6 +510,33 @@ public function setAccept($accept)
...
+            (null !== $value && null !== $currentValue)
+            && (
+                gettype($value) !== gettype($currentValue)
+                && (
+                    !(is_string($value) || is_integer($value))
+                    || !(is_string($currentValue) || is_integer($currentValue))
+                )
+            )
+        ) {
+            throw new \InvalidArgumentException(sprintf(
```
Comment (crouillon) [inline ClassContent/AbstractContent.php]: hum... what happens if the default parameter is null (parameter_name: ~) ?

Comment (eric-chau) [inline ClassContent/AbstractContent.php]: The condition won't be satisfied thanks to this condition: ``(null !== $value && null !== $currentValue)``

**Comment by crouillon** [comment]
:+1:  thx @eric-chau

**Comment by mickaelandrieu** [comment]
:+1:  you can merge, thank you !

**Pull request merged**